unstructured/partition/pdf.py

```diff
@@ -464,7 +464,52 @@ def convert_pdf_to_images(
             yield image
 
 
-def add_pytesseract_bbox_to_elements(elements, bboxes, width, height):
+def _get_element_box(
+    element: Text,
+    boxes: List[str],
+    box_idx: int,
+    char_count: int,
+):
+    """Helper function to get the bounding box of an element.
+
+    Args:
+        element
+        boxes (List[str], optional): Defaults to [].
+        box_idx (int, optional): Defaults to 0.
+        char_count (int, optional): Defaults to 0.
+    """
+    min_x = float("inf")
+    min_y = float("inf")
+    max_x = 0
+    max_y = 0
+    box_text = ""
+    for box in boxes[box_idx : box_idx + char_count]:  # noqa
+        char, _x1, _y1, _x2, _y2, _ = box.split()
+
+        # pytesseract cleans some characters from text that still appear in the bounding boxes,
+        # so those bounding boxes should be skipped
+        if char not in element.text:
+            char_count += 1
+            continue
+        box_text = box_text + char
+
+        x1, y1, x2, y2 = map(int, [_x1, _y1, _x2, _y2])
+        min_x = min(min_x, x1)
+        min_y = min(min_y, y1)
+        max_x = max(max_x, x2)
+        max_y = max(max_y, y2)
```

Comment:
would this be better if we get word level bounding boxes so it can save us operations? `pytesseract.image_to_data` returns a table with bounding boxes for each "word" (spans bounded by white space). What's more the return can be a pd dataframe so we can do vector match with the table instead of running this in a loop with string operations, type casing, and comparison operations in python (which is more costly than in c++ where tesseract runs)

Reply:
That sounds like a great idea. Didn't realize pytesseract would return words

Reply:
Should this also be a follow-up issue or should I do this before merging?

Reply:
a follow up for sure